extensions/psx-pdfinfo.el

```diff
@@ -39,15 +39,9 @@
 
 (require 'p-search)
 
-(p-search-def-field 'pdf-title 'text :weight 3)
 (p-search-def-field 'pdf-subject 'text :weight 3)
-(p-search-def-field 'pdf-producer 'text :weight 3)
```

Comment:
This one is still referenced below as a textual item.  Producer is an odd enough field that it should be kept on its own IMHO.

Reply:
Oh yeah, I removed this particular line because it was duplicated [below](https://github.com/zkry/p-search/pull/65/files#diff-fe0d33782f29159cf66e151813268e1a3a27be4a430e438b01d6208d731c0e4eL48).